### PR TITLE
Support sliced files in ABS table output mapping 

### PR DESCRIPTION
**Pull request by odinuv**
https://keboola.atlassian.net/browse/PS-1647

Checks if the source is a directory and if so, it loads the directory contents as slices.

### CODE REVIEW

File: tests/Writer/AbsWriterWorkspaceTest.php [hunk 1]
```diff
@@ -97,48 +98,155 @@ function () {
...
+        $this->assertCount(1, $tables);
+        $this->assertEquals('out.c-output-mapping-test.table1a', $tables[0]['id']);
+        $job = $this->clientWrapper->getBasicClient()->getJob($jobIds[0]);
+        $this->assertEquals('out.c-output-mapping-test.table1a', $job['tableId']);
+        $this->assertEquals(false, $job['operationParams']['params']['incremental']);
+        $this->assertEquals(['Id', 'Name'], $job['operationParams']['params']['columns']);
+        $data = $this->clientWrapper->getBasicClient()->getTableDataPreview('out.c-output-mapping-test.table1a');
+        $rows = explode("\n", trim($data));
+        sort($rows);
+        // column name is lowercase because of https://keboola.atlassian.net/browse/KBC-864
```
Comment (pivnicek) [inline tests/Writer/AbsWriterWorkspaceTest.php:144]: I think this comment only applies for RS

File: tests/Writer/AbsWriterWorkspaceTest.php [hunk 2]
```diff
@@ -97,48 +98,155 @@ function () {
...
+        $this->assertEquals('out.c-output-mapping-test.table1a', $tables[0]['id']);
+        $this->assertNotEmpty($jobIds[0]);
+        $job = $this->clientWrapper->getBasicClient()->getJob($jobIds[0]);
+        $this->assertEquals('out.c-output-mapping-test.table1a', $job['tableId']);
+        $this->assertEquals(false, $job['operationParams']['params']['incremental']);
+        $this->assertEquals(['first column', 'second column'], $job['operationParams']['params']['columns']);
+        $data = $this->clientWrapper->getBasicClient()->getTableDataPreview('out.c-output-mapping-test.table1a');
+        $rows = explode("\n", trim($data));
+        sort($rows);
+        // column name is lowercase because of https://keboola.atlassian.net/browse/KBC-864
```
Comment (pivnicek) [inline tests/Writer/AbsWriterWorkspaceTest.php:196]: likewise leftover cp

**Review by pivnicek — APPROVED** [review]
looks great!  there's just a stray comment that got left over from thee cp

File: tests/Writer/AbsWriterWorkspaceTest.php (hunk 1 above)
Comment (odinuv) [reply tests/Writer/AbsWriterWorkspaceTest.php:144]: fixed in https://github.com/keboola/output-mapping/pull/72

File: tests/Writer/AbsWriterWorkspaceTest.php (hunk 2 above)
Comment (odinuv) [reply tests/Writer/AbsWriterWorkspaceTest.php:196]: fixed in https://github.com/keboola/output-mapping/pull/72

**Pull request merged**